cornflow-dags/DAG/rostering/README.rst

```diff
@@ -43,4 +48,15 @@ Parameters
   - Opening days: the number of days that the work center opens, the first day is always considered to be a Monday.
   - Slot length: the length of each time slot in minutes.
   - Starting date: the first day that has to be solved.
-  - Starting hour: the hour the work center opens. 
\ No newline at end of file
+  - Starting hour: the hour the work center opens.
+
+- Requirements: table indicating which requirements should be complied and which not
+
+    - Skills: true or false
+    - Skills employees: true or false
+    - Skill demand: true or false
```

Comment:
Those 3 actually correspond to only one constraint, so there should be only one parameter managing this.